Formula/purescript.rb

```diff
@@ -22,6 +22,11 @@ class Purescript < Formula
   uses_from_macos "zlib"
 
   def install
+    # Use ncurses in REPL, providing an improved experience when editing long
+    # lines in the REPL.
+    # See https://github.com/purescript/purescript/issues/3696#issuecomment-657282303.
+    inreplace "stack.yaml", "terminfo: false", "terminfo: true"
```

Comment:
Can we pass this configuration via a command-line argument instead?

Reply:
Not as far as I know, no.